docs/src/booklet/6-solvers.md

```diff
@@ -0,0 +1,21 @@
+# QUBO Solvers
+
+## Solvers, Annealers & Samplers
+`ToQUBO`'s main goal is to benefit from non-deterministic samplers, especially *Quantum Adiabatic* devices and other *Annealing* machines. A few `MOI`-compliant interfaces for annealers and samplers are bundled within `ToQUBO` via the `Anneal.jl` submodule and package prototype. Some of them are presented below.
```

Comment:
There are no Quantum Adiabatic devices, as the Quantum Adiabatic Algorithm is an idealized method, which is not implementable. The samplers can also be deterministic (think of all ones or complete samplers). I would rewrite this as follows:
'ToQUBO''s (Are we calling the package without the .jl? If so, we should remove the .jl for Anneal below and maybe add a hyperlink to its repository) main goal is to make use of parameterized stochastic optimization solvers, particularly those relying on non-conventional hardware such as *Quantum Annealing* and other *Ising Machines*. A few...

Reply:
Done!